CHANGELOG.md

```diff
@@ -65,15 +65,20 @@ To be released on Jun 30, 2020.
 - [#1605][1605] Add to `fiddling.hexdump` a way to suppress the total at the end
 - [#1613][1613] Permit `--password` for `pwn template`
 - [#1564][1564] Fix `asm()` and `disasm()` for PowerPC64, MIPS64, Sparc64
+- [#1606][1606] Fix `asm()` and `disasm()` for MSP430, S390
 
 [1576]: https://github.com/Gallopsled/pwntools/pull/1576
 [1584]: https://github.com/Gallopsled/pwntools/pull/1584
 [1592]: https://github.com/Gallopsled/pwntools/pull/1592
 [1593]: https://github.com/Gallopsled/pwntools/pull/1593
 [1601]: https://github.com/Gallopsled/pwntools/pull/1601
 [1605]: https://github.com/Gallopsled/pwntools/pull/1605
+<<<<<<< HEAD
```

Comment:
Looks like you committed conflict

Reply:
Whoops, I only noticed one of the two conflicts. Fixed now. Also rebased on top of `dev` again :+1: